clients/src/main/resources/common/message/ConsumerGroupHeartbeatRequest.json

```diff
@@ -18,15 +18,15 @@
   "type": "request",
   "listeners": ["zkBroker", "broker"],
```

Comment:
@dajac Does zk broker support this request?

Reply:
Nope it does not. It returns the unsupported version error. The zk stuff will be removed in 4.0 anyway.

Reply:
I will remove `zkBroker` 😺

Reply:
@frankvicky thanks for the quick acknowledgment, but this is unrelated to this PR. We can handle the cleanup in a follow-up.

Reply:
I see, I will revert it shortly